haddock3.sh

```diff
@@ -5,7 +5,10 @@ HADDOCK3_DIR="$HOME/repos/haddock3"
 
 # shellcheck source=/dev/null
 source "$HADDOCK3_DIR/venv/bin/activate" || exit
-
+# if your haddock3 installation uses conda
```

Comment:
For conda use I would rather define:

```
# if your haddock3  installation uses conda
##source $HOME/miniconda3/etc/profile.d/conda.sh
##conda activate haddock3
```

The `bin/activate` command if I am correct add the conda initialisation to your .cshrc / .profile files.

Reply:
This is what I used for the workshop on the Fugaku supercomputer (and for benchmarking on Snellius)